src/main/java/pl/wip/wipplatformbe/enums/Badge.java

```diff
@@ -0,0 +1,6 @@
+package pl.wip.wipplatformbe.enums;
+
+public enum Badge {
+    // define in future
```

Comment:
It's better to use TODO: <message> for this because it's easier to find those parts of code in the future. [Read this](https://www.jetbrains.com/help/idea/using-todo.html)

Reply:
True